cmd/olm/manager.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/operators"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/feature"
+)
+
+var log = ctrl.Log.WithName("manager")
+
+func Manager() (ctrl.Manager, error) {
+	ctrl.SetLogger(zap.Logger(true))
+	setupLog := log.WithName("setup")
+
+	// Setup a Manager
+	setupLog.Info("configuring manager")
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{MetricsBindAddress: "0"}) // TODO(njhale): Enable metrics on non-conflicting port (not 8080)
+	if err != nil {
+		return nil, err
+	}
+
+	// Setup a new controller to reconcile Operators
+	setupLog.Info("configuring controller")
```

Comment:
I would remove these info lines or change to debug

Reply:
I bumped the log level to 4, so we shouldn't see these unless `-v=4` at minimum.